.gitignore

```diff
@@ -90,3 +90,4 @@ NuGetExeBuild/
 NuGetRefBuild/
 Packages/
 NuGetTaskBuild/
+build/
```

Comment:
Maybe we should move those two files out of the build folder. Or make everything compile into artifacts or something which can be ignored?